File: 코딩 기초 트레이닝/문자열이 몇 번 등장하는지 세기-p181871/문자열이 몇 번 등장하는지 세기.js
// https://school.programmers.co.kr/learn/courses/30/lessons/181871

function solution(myString, pat) {
    var answer = 0;

    for(let i = 0; i <= myString.length - pat.length ; i++) {
        if(myString.slice(i, i+pat.length) === pat) {
            answer += 1;
        }
    }

    return answer;
}

File: 코딩 기초 트레이닝/문자열이 몇 번 등장하는지 세기-p181871/문자열이 몇 번 등장하는지 세기.py
# https://school.programmers.co.kr/learn/courses/30/lessons/181871

def solution(myString, pat):
    answer = 0

    for i in range(len(myString) - len(pat) +1):
        if myString[i: i+len(pat)] == pat:
            answer += 1


    return answer
